examples/animations.amp.html

```diff
@@ -20,17 +20,20 @@
   "iterations": 5,
   "fill": "both",
   "direction": "alternate",
+  "easing": "steps(1,end)",
```

Comment:
This is actually overridden in all animations, right? Maybe remove the easing from one of them so that it is clear that this is the one used if no other is defined?